GooglePlayInstant/Editor/QuickDeploy/LoadingScreenGenerator.cs

```diff
@@ -200,5 +230,21 @@ private static Vector2 GetPreImportTextureDimensions(Texture2D texture)
             tempTexture.LoadImage(imageBytes);
             return new Vector2(tempTexture.width, tempTexture.height);
         }
+
+        // Visible for testing.
+        internal static Sprite FindReplayButtonSprite()
```

Comment:
Yeah we need to load this using the AssetDatabase because we're generating the scene programmatically. I opted to search for it by name instead of loading it via a direct path, because I can imagine users moving the plugin to a separate folder and breaking that hard path. I'll add a clarifying comment.